libs/particles/effects.ts

```diff
@@ -1,4 +1,6 @@
 namespace particles {
+    let ongoingEffects: ParticleSource[] = [];
```

Comment:
this should be in the game context

Reply:
Not sure I follow, where should I move that?

Reply:
All game state should be stored In the game currentContext since we can push games on top of each other’s.

Reply:
We'll clean this up later. The same applies to the state of multiplayer controllers. They are leaked as global but should be stored in the context.